src/test/java/reposense/commits/CommitInfoAnalyzerTest.java

```diff
@@ -65,6 +66,17 @@ public void analyzeCommits_eugeneAuthorNoIgnoredCommitsNoDateRange_success() {
         Assert.assertEquals(NUMBER_EUGENE_COMMIT, commitResults.size());
     }
 
+    @Test
+    public void analyzeCommits_duplicateAuthorsDuplicateCommits_success() {
+        config.setAuthorList(Arrays.asList(getAlphaAllAliasAuthor(), getAlphaAllAliasAuthor()));
```

Comment:
In this test, we are creating a list that has duplicate authors, so that we can test the `CommitInfoAnalyzer`'s ability to remove duplicate commits.

I would suggest that instead of creating an author list with duplicate authors, we configure the `SINCE_DATE` and `END_DATE` of the `RepoConfiguration` instance to include a time period where such duplicate commits exist.

For example, try running RepoSense with 
```
./gradlew run -Dargs="--repo https://github.com/reposense/testrepo-Alpha.git --since 6/5/2018 --until 9/11/2018 --view --ignore-standalone-config"
```
`testrepo-Alpha` is our test repo in `CommitInfoAnalyzerTest` and this time period has some duplicate commits:


![image](https://user-images.githubusercontent.com/35621759/75375400-e4b68a00-5908-11ea-8553-bcff7761ad7e.png)


We can use this configuration for this test and check if the duplicate commit is removed by `CommitInfoAnalyzer` ?

Reply:
Updated